storage/elasticsearch/build.sbt

```diff
@@ -25,10 +25,10 @@ libraryDependencies ++= Seq(
   "org.apache.predictionio" %% "apache-predictionio-core" % version.value % "provided",
   "org.apache.spark"        %% "spark-core"               % sparkVersion.value % "provided",
   "org.elasticsearch.client" % "elasticsearch-rest-client" % elasticsearchVersion.value,
-  "org.elasticsearch"       %% "elasticsearch-spark-20" % elasticsearchVersion.value
+  "org.elasticsearch"       % "elasticsearch-spark-20_2.11" % elasticsearchVersion.value
     exclude("org.apache.spark", "*"),
   "org.elasticsearch"        % "elasticsearch-hadoop-mr"  % elasticsearchVersion.value,
-  "org.specs2"              %% "specs2"                   % "2.3.13" % "test")
+  "org.specs2"              %% "specs2-core"                   % "2.4.17" % "test")
```

Comment:
Any reason why we choose not the latest version (4.5.1) but 2.4.17?

Reply:
Tests failed to compile under the latest version.